### PR TITLE
Add manual documentation

### PR DESCRIPTION
**Pull request by agodwinrelay**
Added manual documentation to API methods that will be used by the client, as well as package-info files that hold package comments.

### CODE REVIEW

File: app/src/main/java/com/relaypro/app/examples/multi/DeviceInfoWorkflow.java [hunk 1]
```diff
@@ -27,8 +27,7 @@ public void onInteractionLifecycle(Relay relay, InteractionLifecycleEvent lifecy
 
         if (lifecycleEvent.isTypeStarted()) {
             relay.setDeviceName(interactionUri, "optimus prime");
-            relay.setLocationEnabled(interactionUri, true);
```
Comment (cmarcelk) [inline app/src/main/java/com/relaypro/app/examples/multi/DeviceInfoWorkflow.java:30]: Maybe turn this into sayAndWait("location services are " + relay.getLocationEnabled() ? "enabled" : "disabled")

File: sdk/src/main/java/com/relaypro/sdk/Relay.java [hunk 2]
```diff
@@ -763,92 +1005,185 @@ public boolean isGroupMember(String groupNameUri, String potentialMemberNameUri)
...
+     * @return the indoor location of the device.
+     */
     @SuppressWarnings("unused")
     public String getDeviceIndoorLocation( String sourceUri, boolean refresh) {
         DeviceInfoResponse resp = getDeviceInfo( sourceUri, DeviceInfoQueryType.IndoorLocation, refresh);
         return resp != null ? resp.indoor_location : null;
     }
 
+    /**
+     * Returns the batter of a targeted device.
```
Comment (cmarcelk) [inline sdk/src/main/java/com/relaypro/sdk/Relay.java:1137]: typo here.

File: sdk/src/main/java/com/relaypro/sdk/Relay.java [hunk 3]
```diff
@@ -1033,9 +1418,15 @@ public static String getSourceUri(StartEvent startEvent) {
...
+    // The server host name.  Used for updating an access token, starting a workflow through an HTTP trigger,
+    // or retrieving information on a device through the server.
     String serverHostname = "all-main-pro-ibot.relaysvr.com";
+
+    // The version of the Java SDK.
     String version = "relay-sdk-java/2.0.0";
-    String auth_hostname = "auth.relaygo.com";
+
+    // The auth hostname used in URL when granting a new access token.
+    String auth_hostname = "auth.relayjava.com";
```
Comment (cmarcelk) [inline sdk/src/main/java/com/relaypro/sdk/Relay.java:1429]: I think this change was unintentional.

File: app/src/main/java/com/relaypro/app/examples/multi/DeviceInfoWorkflow.java (hunk 1 above)
Comment (agodwinrelay) [reply app/src/main/java/com/relaypro/app/examples/multi/DeviceInfoWorkflow.java:30]: Good idea - fixed in latest commit.

**Review by cmarcelk — APPROVED** [review]

**Pull request merged**